assets/css/WorkflowEditor/Report/_mixins.scss

```diff
@@ -24,7 +24,7 @@
     justify-content: flex-end;
 
     button {
-      margin: 0.5em 0 0 0.5em;
+      margin: 1em 0 1em 1em;
```

Comment:
Ok I switched to padding while keeping the distance consistent above and below .buttons

Reply:
@pierreconti Please answer my question:

> On hover, the top buttons are flush with the top of the left-hand blue bar; but the bottom buttons aren't flush with the bottom, because of this new `margin-bottom`. Is that intentional?